src/seedu/addressbook/commands/Command.java

```diff
@@ -67,4 +68,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isAutoListEnabled(){ return autoListEnabled; }
```

Comment:
1. Please check the coding standards on how to write methods with a single line of code.
2. Need space between `) {`